operator_ui/src/pages/JobRuns/Index.js

```diff
@@ -65,9 +67,18 @@ const renderDetails = (props, state, handleChangePage) => {
   }
 }
 
+const fetchRuns = async (page, size) => {
+  const response = await v2.runs.getAllJobRuns({ page, size })
+
+  return response.data.map((run) =>
+    transformPipelineJobRun(run.pipelineSpec.ID)(run),
```

Comment:
`transformPipelineJobRun` takes the job id, not the pipeline spec id.

It should also call `run.attributes.pipelineSpec...`

The API still does not return the job id at the moment